server/src/test/java/com/qubole/quark/serverclient/client/DbEndToEndTest.java

```diff
@@ -0,0 +1,41 @@
+package com.qubole.quark.serverclient.client;
+
+import com.qubole.quark.serverclient.server.Main;
+import org.flywaydb.core.Flyway;
+import org.junit.BeforeClass;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.sql.SQLException;
+
+/**
+ * Created by adeshr on 3/3/16.
+ */
+public class DbEndToEndTest extends EndToEndTest {
+
+  public static String dbUrl = "jdbc:h2:mem:DbTpcds;DB_CLOSE_DELAY=-1";
+  static {
+    String[] args = new String [1];
+    args[0] = JsonEndToEndTest.class.getResource("/dbCatalog.json").getPath();
+    main = new Main(args);
+    h2Url = "jdbc:h2:mem:DbServerTpcdsTest;DB_CLOSE_DELAY=-1";
+    cubeUrl = "jdbc:h2:mem:DbServerTpcdsCubes;DB_CLOSE_DELAY=-1";
+    viewUrl = "jdbc:h2:mem:DbServerTpcdsViews;DB_CLOSE_DELAY=-1";
+  }
+
+  @BeforeClass
+  public static void setUp() throws SQLException, IOException, URISyntaxException,
+      ClassNotFoundException {
+
+    new Thread(main).start();
+
+    Flyway flyway = new Flyway();
```

Comment:
Hmm - There is something fishy going on. When I ran the server, I noticed that the migrations did not run. This seems to be same symptom. Ideally server should run it on start up.